packages/drafts-realm/event.gts

```diff
@@ -0,0 +1,251 @@
+        border: 1px solid var(--boxel-form-control-border-color);
+        padding: var(--boxel-sp-xs) 0 var(--boxel-sp-xs) var(--boxel-sp-sm);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class Isolated extends Component<typeof Event> {
+  <template>
+    <div class='row'>
+      <div class='column'>
+        <FieldContainer @label='Subject' @tag='label' class='field'>
+          <@fields.subject />
+        </FieldContainer>
+        <FieldContainer @label='Location' @tag='label' class='field'>
+          <@fields.location />
+        </FieldContainer>
+        <FieldContainer @label='Start' @tag='label' class='field'>
+          <div class='inner-row'>
+            <@fields.startDate />
+            <div class='divider' />
+            <@fields.startTime />
+          </div>
+        </FieldContainer>
+        <FieldContainer @label='End' @tag='label' class='field'>
+          <div class='inner-row'>
+            <@fields.endDate />
+            <div class='divider' />
+            <@fields.endTime />
+          </div>
+        </FieldContainer>
+        <FieldContainer @label='Event type' @tag='label' class='field'>
+          <@fields.description />
+        </FieldContainer>
+        <FieldContainer @label='Description' @tag='label' class='field'>
+          <@fields.description />
+        </FieldContainer>
+        <FieldContainer
+          @label='Is reminder set'
+          @tag='label'
+          class='field column'
+        >
+          <@fields.isReminderSet />
+        </FieldContainer>
+      </div>
+      <div class='column'>
+        <FieldContainer @label='Assigned to' @tag='label' class='field'>
+          <@fields.assignee />
+        </FieldContainer>
+      </div>
+    </div>
+    <div class='row'>
+      <FieldContainer @label='Location' @tag='label' class='field'>
+        <@fields.location />
+      </FieldContainer>
+      <div class='column' />
+    </div>
+    <style>
+      .row {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        padding: 0 calc(1.25rem * 1.333);
+        margin-top: 20px;
+      }
+      .inner-row {
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+      }
+      .divider {
+        width: 20px;
+      }
+      .column {
+        width: 50%;
+      }
+      .column > label {
+        margin-bottom: 20px;
+      }
+    </style>
+  </template>
+}
+
+export class Event extends CardDef {
+  static displayName = 'Event form';
+  @field subject = contains(StringCard);
+  @field location = contains(StringCard);
+  @field assignee = linksTo(PersonCard);
+  @field startDate = contains(DateField);
+  @field endDate = contains(DateField);
+  @field startTime = contains(TimeCard);
+  @field endTime = contains(TimeCard);
+  @field eventType = contains(StringCard);
+  @field description = contains(TextAreaCard);
+  @field isReminderSet = contains(BooleanField);
```

Comment:
How are you planning on implementing setting a reminder? Conventionally, the reminder would only ever fire when the card is actually running. You probably need a service worker to fire a reminder for the card if it’s not running. (And that all assumes the browser is actually running).

Reply:
If you don’t plan on triggering a reminder, then what’s the point of this field?

Reply:
This field was removed. We will re-include it when the time comes to consider "firing the reminder"